string[] listInputDescription
string[] listAssociatedChannel
float32[] listActivationValue
float32 eventActivationValue